src/util.js

```diff
@@ -481,6 +481,30 @@ export default {
   },
 
   /**
+   * Get native Web Cryptography api for old browsers only, including legacy
+   * implementations of the spec e.g IE11 and Safari 8/9. The default
+   * configuration is to use the api when available. But it can be deactivated
+   * with config.use_native
+   * @return {Object}   The SubtleCrypto api or 'undefined'
+   */
+  getWebCryptoPrefixed: function() {
+    if (!config.use_native) {
+      return;
+    }
+
+    var _self = this.getGlobal();
+    if (typeof _self === 'undefined') {
+      return;
+    }
+
+    if (typeof _self.crypto !== 'undefined') {
+      return _self.crypto.webkitSubtle;
```

Comment:
Ok. I added a check to make sure the standard api is detected. If it is available `util. getWebCryptoPrefixed()` will return undefined.

https://github.com/openpgpjs/openpgpjs/blob/issue/449/src/util.js#L483-L509